projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -1411,10 +1411,20 @@
     font-weight: normal;
 
     .datagrid-filter-close-wrapper {
-      text-align: right;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      gap: #{tokens.$cds-global-space-3};
```

```suggestion
      gap: tokens.$cds-global-space-3;
```

Is interpolation needed here? Check the `margin-bottom` one too.

---

corrected